plugins/katello/3.15/user_guide/remote_databases/index.md

```diff
@@ -24,6 +24,14 @@ There are two ways to deploy Foreman with remote databases:
     * restore the DBs on remote servers
     * run foreman-installer with right parameters pointing to the databases. It re-configures the databases and start all the services with new DB locations
 
+### Important note:
+
+Pulpcore installs a separate PostgreSQL database and lacks support for external databases. This support is introduced in Katello 3.16.
+
+Foreman and Pulpcore require PostgreSQL 10 or newer. Databases should be upgraded to avoid conflicts. It is recommended to update to version 12.
```

Comment:
In other files it uses bullet points, here it doesn't. Feels a bit inconsistent.

Reply:
@ekohl After I remove the second line, we no longer have "important notes" but an "important note: I wonder if that formatting should change too.